device_backends/LogicalNameMapping/src/LNMMathPlugin.cc

```diff
@@ -77,6 +77,11 @@ namespace ChimeraTK::LNMBackend {
       _mainValueWrittenAfterOpen = false;
       _allParametersWrittenAfterOpen = false;
     }
+
+    // consume all entries from the push-type parameter queues, to prevent old exceptions from being thrown again
+    if(_hasPushParameter) {
+      getFormulaHelper(backend)->clearParameterQueues();
```

Comment:
When you first showed me this you mentioned that this might unnecessarily instantiate a formula helper. Do you think it's worth putting this information into a comment? With the improved solution without queues this section will go out anyway.